eng/scripts/Detect-ApiChanges.ps1

```diff
@@ -0,0 +1,129 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [array] $ArtifactList,
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $APIViewUri,
+  [Parameter(Mandatory=$True)]
+  [string] $buildId,
+  [Parameter(Mandatory=$True)]
+  [string] $repoName,
+  [Parameter(Mandatory=$True)]
+  [string] $PullRequestNumber,
+  [Parameter(Mandatory=$True)]
+  [string] $CommitSha,
+  [string] $TargetBranch = ("origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" -replace "refs/heads/")
+)
+
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+
+function Submit-Request($filePath, $packageName)
+{
+    $repoName = $RepoFullName
+    if (!$repoName) {
+        $repoName = "azure/azure-sdk-for-$LanguageShort"
+    }
+    $reviewFileName = "$($packageName)_$($LanguageShort).json"
+    $query = [System.Web.HttpUtility]::ParseQueryString('')
+    $query.Add('artifactName', $ArtifactName)
+    $query.Add('buildId', $BuildId)
+    $query.Add('filePath', $filePath)
+    $query.Add('commitSha', $CommitSha)
+    $query.Add('repoName', $repoName)
+    $query.Add('pullRequestNumber', $PullRequestNumber)
+    $query.Add('packageName', $packageName)
+    $query.Add('language', $LanguageShort)
+    $query.Add('codeFile', $reviewFileName)
```

Comment:
Are these the only real changes to this script?

Reply:
yes.  Changing this over to common script is easier for API change detect in PR request. I will change common script as soon as other language starts creating and publishing API code file artifact.

Reply:
I will actually check if I can make a change in common script itself for this use case and pass these new params conditionally.

Reply:
Removed this file.  I have updated common file to pass extra parameter conditionally if APIview code file exists. This was doable since we reused existing REST API to check API changes.